docs-gen/Makefile

```diff
@@ -1,11 +1,5 @@
-build: clean
+build: clean install-npm-dependencies
 	hugo --minify
-	@# Workaround for https://github.com/google/docsy/issues/350 and https://github.com/gohugoio/hugo/issues/7714
-	$(eval URL_PATH = $(shell grep baseURL config.toml | sed "s/'//g" |  cut -d '/' -f 4))
-	@ if [ ! -z "$(URL_PATH)" ]; \
-	  then \
-	    sed -i.bak "s/background-image:url(/background-image:url(\/$(URL_PATH)/g" ../docs/index.html && rm ../docs/index.html.bak; \
-      fi
```

Comment:
we no longer need to do this as we're not setting `canonifyurls = true` to true anymore, meaning the background image already has the correct path